src/angular/controllers/ctrl-my-widgets-collaboration.test.js

```diff
@@ -0,0 +1,41 @@
+describe('MyWidgetsCollaborationController', () => {
```

Comment:
Since the entire My Widgets page and all subsequent files were rewritten in React, this file isn't being used any more. I'll add another commit to remove it.